lib/datasource/helm/index.js

```diff
@@ -0,0 +1,84 @@
+const yaml = require('js-yaml');
+const got = require('../../util/got');
+
+module.exports = {
+  getPkgReleases,
+};
+
+async function getPkgReleases({ lookupName, helmRepository }) {
```

Comment:
Instead of this custom new `helmRepository` field, can we reuse our existing `registryUrls` concept?

Reply:
There can be multiple repositories in a single `requirements.yaml` file, and each dependency is associated with it's own repository. So if `registryUrls` is used, we will have to pass `helmRepository` with every dependency anyway in order to identify which registry url to use. 

Getting `index.yaml` for every registry in `registryUrls` and searching for `lookupName` in them without `helmRepository` wouldn't work, because packages can be duplicated in repositories, and there is no guarantee that the right one will be found. Also it would be more expensive.